jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibPluginIntegrationTest.java

```diff
@@ -350,4 +362,58 @@ public void testDockerContext() throws IOException, InterruptedException {
               "Export Docker context failed because cannot clear directory"));
     }
   }
+
+  @Test
+  public void testMultiProject() {
+    BuildResult buildResult =
+        multiprojectTestProject.build(
+            "clean", ":a_packaged:" + JibPlugin.DOCKER_CONTEXT_TASK_NAME, "--info");
+
+    BuildTask classesTask = buildResult.task(":a_packaged:classes");
+    BuildTask jibTask = buildResult.task(":a_packaged:" + JibPlugin.DOCKER_CONTEXT_TASK_NAME);
+    Assert.assertNotNull(classesTask);
+    Assert.assertEquals(TaskOutcome.SUCCESS, classesTask.getOutcome());
+    Assert.assertNotNull(jibTask);
+    Assert.assertEquals(TaskOutcome.SUCCESS, jibTask.getOutcome());
+    Assert.assertThat(
+        buildResult.getOutput(), CoreMatchers.containsString("Created Docker context at "));
+  }
+
+  @Test
+  public void testBuildTar_entrypoint() throws IOException {
```

Comment:
If we do choose only one of the tasks to test, I would lean towards `jib` (build to registry) since it is the main functionality of Jib.

Reply:
Maybe this method should just be called `testCustomEntrypoint` then?